server/protocols/jmap/src/main/java/org/apache/james/jmap/model/OldKeyword.java

```diff
@@ -170,6 +171,14 @@ public Flags applyToState(Flags currentFlags) {
         if (shouldMessageBeMarkSeen) {
             newStateFlags.add(Flags.Flag.SEEN);
         }
+        Arrays.stream(currentFlags.getUserFlags())
```

Comment:
No test for user flags ?

Reply:
Added ;-)

Reply:
Sorry I added it concurrently to your remark...